packages/monorepo-scripts/package.json

```diff
@@ -28,7 +28,7 @@
     "pre-commit": "monorepo-tools pre-commit"
   },
   "dependencies": {
-    "@babel/core": "7.7.7",
+    "@babel/core": "7.12.0",
```

Comment:
This is long overdue! This brings us better performance, enhanced typescript support and ES2020 support! 🥳.

Tested this locally, and everything works